src/main/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommand.java

```diff
@@ -70,7 +70,6 @@ public CommandResult execute(Model model) throws CommandException {
         OrderList orderList = personToAddUnder.getOrderList();
         String feedbackToUser = orderList.contains(toAdd) ? MESSAGE_DUPLICATE_ORDER_WARNING : "";
         orderList.add(toAdd);
-        personToAddUnder.resetFilteredOrderList();
```

Comment:
u need to undo this since this is from previous version.

Reply:
Updated.